core-feature-pack/src/main/resources/content/bin/init.d/wildfly-init-debian.sh

```diff
@@ -108,6 +109,7 @@ else
 	if [ -z "$JBOSS_HOST_CONFIG" ]; then
 		JBOSS_HOST_CONFIG=host.xml
 	fi
+	JBOSS_MARKERFILE="$JBOSS_HOME/domain/tmp/jboss-as-startup-marker"
```

Comment:
These two script files use tabs in new lines, do you want me to reformat them all?